### ISSUE TITLE
cdc: how to get correct Version when debug cdc

### ISSUE THREAD

**Issue by Benevor** [pingcap/tiflow#9221]
### Before asking a question, make sure you have

- [X] Searched existing Stack Overflow questions
- [X] Googled your question
- [X] Searched open and closed [GitHub issues](https://github.com/pingcap/tiflow/issues?q=is%3Aissue)
- [X] Read the [documentation](https://docs.pingcap.com/tidb/stable)

### What is your question?

I want to debug ticdc in ubuntu environment. When I start debugging from cmd/cdc, ticdc works correctly. But when I created the changefeed, `panic` appeared in `creatorVersion := semver.New(SanitizeVersion(g.version))`, so I tried to skip the version check here and successfully created the changefeed. Of course, this turned out to be a stupid move, because the changefeed still crashed when running. But at this time, this wrong changefeed already exists in etcd, so whenever I debug ticdc, this changefeed will run, causing the system to crash. 
So I wonder where etcd's data is stored when TiCDC uses etcd? Or is there any other command to clear the changefeed information? thanks for your answers. (ps. there is no /var/lib/etcd folder on my system)

**Comment by Benevor** [comment]
I have found the floder: /tmp/cdc_data

**Comment by sdojjy** [comment]
@Benevor 
1. for the version check, I think it's better to modify the version.go https://github.com/pingcap/tiflow/blob/72afa810214abdbb419d935c5e4df65213f6c524/pkg/version/version.go#LL27C25-L27C25 , set the `ReleaseVersion ` with a valid value like `v7.1.0`.  
2. you can use `cdc  cli unsafe  reset ` to clear all cdc metadata and changefeeds,  or you can use `etcdctl del' to delete the related changefeed keys in etcd ,then restart cdc cluster. 
3. for the `panic` things, we are working on it. invalid changefeed will not panic the CDC server in a future release.

**Comment by Benevor** [comment]
Thank you for your answer!
1. For the version check, I tried to write the version number in the code yesterday. When debugging now, everything works fine.
2. For etcd, I'm still not sure where its data is in the file system, it doesn't seem to be `/tmp/cdc_data`. I am not very familiar with the use of etcd. During the debugging process, I saw a prefix like `/tidb/cdc`. That doesn't mean such a path exists in the filesystem, does it?
